packages/frontend/src/components/modal/TxStatusModal.tsx

```diff
@@ -49,16 +53,45 @@ function TxStatusModal(props: Props) {
       />
 
       <Box display="flex" alignItems="center" className={styles.txStatusInfo}>
-        <Typography variant="body1">
-          {(tx && tx.token && timeEstimate) ? (
-            <em>
-              Your transfer will arrive at the destination in <strong>{timeEstimate}</strong>{' '}
-              after your transaction is confirmed.
-            </em>
+        <Box>
+          {(tx && tx.token && medianTimeEstimate) ? (
+            <Box margin="0 auto" maxWidth="32rem" paddingLeft={3} paddingRight={3}>
+              <Typography variant="body2" color="textSecondary">
+                Your {tx.token._symbol ? tx.token._symbol : "transfer"}{' '}
+                will arrive{' '}
+                {
+                  tx.destNetworkName
+                  ? `on ${networkSlugToName(tx.destNetworkName)}` 
+                  : "at the destination"
+                }
+                <strong>{' '}~
+                  {(medianTimeEstimate !== null && medianTimeEstimate > 0) 
+                  ? (medianTimeEstimate + " " + `${pluralize('minute', medianTimeEstimate)}`) 
+                  : (fixedTimeEstimate + " " + pluralize('minute', fixedTimeEstimate))}
+                </strong>{' '}
+                after the transaction is confirmed.
+                { percentileTimeEstimate !== null && percentileTimeEstimate > 0 && (
+                  <>
+                    <br />
+                    <br />
+                    Only 10% of recent transfers took more than {percentileTimeEstimate + " " + pluralize('minute', percentileTimeEstimate)}.
+                  </>
+                )}
+              </Typography>
+              { medianTimeEstimate > fixedTimeEstimate * 1.5 &&
+                <>
+                  <br />
+                  <Typography variant="body2" color="textSecondary" style={{ fontStyle: 'italic' }}>
+                    This estimate is higher than usual and may not reflect current speeds.
+                  </Typography>
+                </>
+              }
+            </Box>
           ) : (
-            <em>This may take a few minutes</em>
+            <Typography variant="body1" style={{ fontStyle: 'italic' }}>This may take a few minutes</Typography>
```

Comment:
minor suggestion: use `<em>` for italic text instead of inline style for simplicity and semantics

```suggestion
<Typography variant="body1"><em>This may take a few minutes</em></Typography>
```